pkg/runner/local_runner.go

```diff
@@ -3,20 +3,38 @@ package runner
 import (
 	"context"
 	"errors"
+	"fmt"
 	"os/exec"
 	"path/filepath"
+	"strings"
 
 	"github.com/buildbarn/bb-remote-execution/pkg/proto/runner"
 	"github.com/buildbarn/bb-storage/pkg/filesystem"
 	"github.com/buildbarn/bb-storage/pkg/filesystem/path"
 	"github.com/buildbarn/bb-storage/pkg/util"
+	"golang.org/x/sys/unix"
 
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 	"google.golang.org/protobuf/types/known/anypb"
 	"google.golang.org/protobuf/types/known/emptypb"
 )
 
+// Mount information for the special filesystems
+// to mount in the input root when using 'chroot'.
+var (
+	procMountpointComponent = path.MustNewComponent("proc")
+	sysMountpointComponent  = path.MustNewComponent("sys")
+	mounts                  = []struct {
+		mountpoint path.Component
+		fstype     string
+		source     string
+	}{
+		{procMountpointComponent, "proc", "/proc"},
```

Comment:
Should this be specified in config?
```proto
repeated MountPointConfiguration mount_points = 123;
```

In main.go:
```go
for _, mountPoint := range configuration.MountPoints {
     runner = NewMountingRunner(runner, mountPoint)
}
```
?

Reply:
Yepp!